package.json

```diff
@@ -28,8 +28,7 @@
       "@eth-optimism/contracts-bedrock/ds-test",
       "@eth-optimism/contracts-bedrock/forge-std",
       "@eth-optimism/contracts-bedrock/@rari-capital/solmate",
-      "@eth-optimism/contracts-bedrock/@openzeppelin/contracts",
-      "@eth-optimism/contracts-bedrock/@openzeppelin/contracts-upgradeable",
+      "**/@openzeppelin/*",
```

Comment:
```suggestion
      "@eth-optimism/*/@openzeppelin/*",
```
This is an alternative way to do it

Reply:
Can you confirm that this works? I tried this syntax in the past but it didn't seem to work

Reply:
Yup.  Sometimes yarn cache is dumb when you change things like this and resolutions and you hvae to `rm -rf node_modules` to get it to work correctly

Here is a pic of no open zeppelin in root/node_modules
<img width="340" alt="image" src="https://user-images.githubusercontent.com/35039927/207120633-3412d319-a7d6-433a-9e73-7a30cae53bd1.png">
And then contracts bedrock 
<img width="1002" alt="image" src="https://user-images.githubusercontent.com/35039927/207120758-3b86273b-f480-4897-87ef-dcf33015fed6.png">

And then contracts periphery (this version is changing next pr to same as bedrock)
<img width="991" alt="image" src="https://user-images.githubusercontent.com/35039927/207120895-c0d70486-353e-49b2-9541-6c5a88ccc6cd.png">